plugins/modules/aci_system_connectivity_prefs.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, David Neilan (@dneilan-intel) <david.neilan@intel.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_system_connectivity_prefs
+short_description: APIC Connectivity Preferences (mgmt:ConnectivityPrefs)
+description:
+- Manages Alias and Banners on Cisco ACI fabrics.
+options:
+  interface_pref:
+    description:
+    - Interface to use for external connection.
+    type: str
+    choices: [ inband, ooband ]
```

Comment:
Should we add following aliases?
```suggestion
    choices: [ inband, ooband ]
    aliases: [interface_pref, int_pref, external_connection]
```